src/frontend/components/VideoPlayer/index.tsx

```diff
@@ -136,6 +144,14 @@ const VideoPlayer = ({
           ))}
       </video>
       {!player && video.live_state && <WaitingLiveVideo />}
+      {player &&
+        video.live_state &&
+        [liveState.STOPPING, liveState.PAUSED].includes(video.live_state) && (
```

Comment:
I would also use variables here to improve readability.
```suggestion
      {isLiveStarting && <WaitingLiveVideo />}
      {isLivePausedOrStopping && (
```

Reply:
using `useMemo` to compute this value ?

Reply:
Hmm in this case, I don't think that compute these values are heavy so I'll not use `useMemo` here.